### DISCUSSION TITLE
Can batch submitter switch between using singular batch and span batch?

### DISCUSSION THREAD

**Discussion by bap2pecs** [ethereum-optimism/developers#458] [Application Development] [answered]
Let's say the submitter are doing singular batches now. Can he suddenly change the `batch-type` flag and restart the node to use span batches?

If yes, how to config the span size? and can we change span size freely?

**Comment by tynes** [comment, marked as answer]
Yes you can. This is unrelated to the specs as its related to specific implementation configuration. It would be better to ask this question in https://github.com/ethereum-optimism/developers. Closing this topic
